README.md

```diff
@@ -50,22 +50,24 @@ The project is delivered as a docker image that contains our tooling:
 docker pull asecurityteam/sdcli:v1
 ```
 
-With the image installed you call it like:
+With the image installed you call it like (omit the first `--mount` if on Mac):
 
 ```bash
+export cwd=$(pwd)
+export project_path=${cwd#"${GOPATH}/src/"}
 docker run -ti \
-    # Mount and configure SSH inside the container.
+    # If Linux, mount and configure SSH inside the container.
```

Comment:
Is this not applicable to OSX?

Reply:
It wouldn't work for me.  I did not dig in to figure out why.

Reply:
Relevant link for why this doesn't work on Mac: https://github.com/docker/for-mac/issues/410